commcare_connect/reports/views.py

```diff
@@ -401,22 +399,68 @@ def dashboard_stats_api(request):
     filterset = DashboardFilters(request.GET)
 
     # Use the filtered queryset to calculate stats
-    queryset = UserVisit.objects.all()
+    visit_queryset = UserVisit.objects.filter(opportunity__is_test=False)
+    flw_payment_queryset = Payment.objects.filter(opportunity_access__opportunity__is_test=False)
+    org_payment_queryset = Payment.objects.filter(invoice__opportunity__is_test=False)
+    completed_work_queryset = CompletedWork.objects.filter(opportunity_access__opportunity__is_test=False)
     if filterset.is_valid():
-        queryset = filterset.filter_queryset(queryset)
+        visit_queryset = filterset.filter_queryset(visit_queryset)
+        raw_filters = filterset.form.cleaned_data
+        program = raw_filters.get("program")
```

Comment:
nit: we use `program` to mean something different elsewhere in the codebase (a collection of opportunities managed together, see the `programs` directory). It might be clearer if this were `delivery_type` unless we mean `program` in which case it doesn't appear to be filtering correctly.

Reply:
hmm, I was mainly using it to reflect the language on the filters in the front end, which was terminology that I believe I got from Ali. Do you think that was referring to the type of program you're talking about? Or is it a difference between the user-facing terminology and the dev-facing terminology?

Reply:
(I'm going to merge this for now since it'd be great to see the data on the next deploy, but happy to follow up on this if we think we got something wrong)